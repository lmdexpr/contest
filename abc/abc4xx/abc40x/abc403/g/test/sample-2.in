8
105282053
695234822
468007124
120710491
568831200
700753895
765188109
262666319
