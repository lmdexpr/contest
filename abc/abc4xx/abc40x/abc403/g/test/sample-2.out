105282054
105282054
905798931
599798602
995656103
891549225
1652393438
1652393438
